rasa/cli/data.py

```diff
@@ -121,28 +127,28 @@ def _add_data_convert_parsers(
         help="Migrate model configuration between Rasa Open Source versions.",
     )
     migrate_config_parser.set_defaults(func=_migrate_model_config)
-    _add_migrate_config_arguments(migrate_config_parser)
-
-
-def _add_migrate_config_arguments(parser: argparse.ArgumentParser) -> None:
-    parser.add_argument(
-        "-c",
-        "--config",
-        default=DEFAULT_CONFIG_PATH,
-        help="Path to the model configuration which should be migrated",
-    )
-    parser.add_argument(
-        "-d", "--domain", default=DEFAULT_DOMAIN_PATH, help="Path to the model domain"
-    )
-    parser.add_argument(
-        "-o",
-        "--out",
-        type=str,
+    default_arguments.add_config_param(migrate_config_parser)
+    default_arguments.add_domain_param(migrate_config_parser)
+    default_arguments.add_out_param(
+        migrate_config_parser,
         default=os.path.join(DEFAULT_DATA_PATH, "rules.yml"),
-        help="Path to the file which should contain any rules which are created as "
+        help_text="Path to the file which should contain any rules which are created as "
         "part of the migration. If the file doesn't exist, it will be created.",
     )
 
+    convert_responses_parser = convert_subparsers.add_parser(
+        "responses",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        parents=parents,
+        help=(
+            "Convert retrieval intent responses between Rasa Open Source versions. "
+            "Please also run `rasa data convert nlg` to convert responses to the right format."
+        ),
+    )
+    convert_responses_parser.set_defaults(func=_migrate_responses)
+    arguments.set_convert_arguments(convert_responses_parser, data_type="Rasa NLU")
```

Comment:
This argument is used for `add_data_param()`, and in the case of this command, it's mainly stories (because the domain is handled by another argument). I'll change to `stories`